src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function EpisodePreview(){
+
+	return (
+		<form>
+			<button>Close</button>
+			<output>
+				<img 
+					src="path/to/image.jpg" 
+					alt="placeholder description" 
+				/>
+				<video controls src="path/to/video.mp4" />
+			</output>
+			<h2>Episode X</h2>
+			<h1>Name of Episode X</h1>
+			<button>Previous</button>
+			<button>Next</button>
+		</form>
+	);
+}
```

Comment:
A couple comments on the implementation with a `<form>` :

1. Try to group elements in the html where it makes logical sense; remember, at this stage we're almost not thinking about the styling at all - so for the 'close' button to appear by itself at the top is a bit weird; it should be at the bottom with the next / previous
2. If the next / previous control which is the current episode, then the headings should probably be in the `<output>` as well.

That said, as this would be a child of the `<Episode />` component, I think this works better if we come at it less as a form and more of an 'additional content' kinda thing. I'm thinking either:

- an `<aside>` with a kinda `<figure>`-like vibe, or
- a `<details>` - as we're using the thumbnail to hide / reveal the `<video>` and its associated content

if we go for the former, it'd be something like:

```
<aside>
	<a 
		href="path/to/episode/video.mp4"
		aria-controls="episode__1__preview"
	>
		<img src="path/to/episode/thumb.jpg" alt="thumb contents" />
	</a>
	<div 
		id="episode__1__preview"
		aria-hidden="true"
	>
		<header>
			<h1>
				Episode 1: The Whatever Part 1
			</h1>
			<nav>
				<a href="#episode-0" />
				<a href="#episode-2" />
			</nav>
		</header>
		<video>
			// only render the video sources when the figure is activated
		</video>
	</div>
</aside>
```
(only with a bunch more `aria` stuff)

and for the latter:

```
<details>
	<summary>
		<a href="path/to/episode/video.mp4">
			<img src="path/to/episode/thumb.jpg" alt="thumb contents" />
		</a>
	</summary>
	<div id="episode__1__preview">
		<div>
			<h1 id="episode__1__title">
				Episode 1: The Whatever Part 1
			</h1>
			<a href="#episode-0" />
			<a href="#episode-2" />
		</div>
		<video aria-labelledby="episode__1__title">
			// only render the video sources when the figure is activated
		</video>
	</div>
</details>
```

I think either would work, and not sure if that HTML is right, so worth having a ponder on it either way. Your call, cap'n!

Reply:
ok so i set it up to use the aside approach you outlined.

I have the thumbnail link toggling the hidden state for the preview overlay.
What's missing is the ability to turn on/off hidden for prev/next preview overlays, and updating the episode breakdown (that is obscured by the active overlay).
Thoughts on the best way to implement that?

Any other thoughts on it much appreciated. Bit confused 😄